UI/cypress/e2e/complaints-page.js

```diff
@@ -0,0 +1,53 @@
+import { Given, Then, When } from "@badeball/cypress-cucumber-preprocessor";
+
+Given("the ‘contact us’ page is displayed", () => {
```

Comment:
Could this be renamed "a user has arrived on the report issues page" and moved to `pages.js` so it can be shared?

Reply:
Step moved to pages.js